### DISCUSSION TITLE
Issues formatting document

### DISCUSSION THREAD

**Discussion by mriostamez** [serkonda7/vscode-vba#114] [Q&A] [answered]
Hello there!
I installed the extension v0.10.2 on VS Code 1.87.2, but I'm unable to format the code.
At ALT+SHIFT+F VS Code reports, ```There is no formatter for 'vba' files installed```.

**Is there something I can correct to make this work?**

Thanks!

**Comment by serkonda7** [comment, marked as answer]
Hello, formatting is yet to be implemented.
Please also take a look at #90

**Reply by mriostamez** [reply]
Thanks @serkonda7! Is there an estimated date for this release?
Is there another extension you would recommend that does this today?

**Reply by serkonda7** [reply]
No estimate. I already had to pause my work on code export due to more urgent stuff. And formatting is not my priority

I'm not aware of a extension that provides formatting but you might search for standalone tools.

**Reply by mriostamez** [reply]
Got it, thanks!
